### PR TITLE
v5 Docs

### PR DESCRIPTION
**Pull request by eliperelman**
Updating docs for v5. This will not land before v5 does.

### CODE REVIEW

File: docs/middleware/neutrino-middleware-loader-merge/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+#### npm
+
+```bash
+❯ npm install --save neutrino-middleware-loader-merge
+```
+
+## Usage
+
+`neutrino-middleware-hot` can be consumed from the Neutrino API, middleware, or presets. Require this package
```
Comment (helfi92) [inline docs/middleware/neutrino-middleware-loader-merge/README.md]: s/`neutrino-middleware-hot`/`neutrino-middleware-loader-merge`/

File: docs/middleware/neutrino-middleware-minify/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Requirements
+
+- Node.js v6.9+
+- Yarn or npm client
+- Neutrino v5
+
+## Installation
+
+`neutrino-middleware-babili` can be installed via the Yarn or npm clients.
```
Comment (helfi92) [inline docs/middleware/neutrino-middleware-minify/README.md]: s/`neutrino-middleware-babili`/`neutrino-middleware-minify`/

File: docs/middleware/neutrino-middleware-named-modules/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# Neutrino Named Modules Middleware
+[![NPM version][npm-image]][npm-url] [![NPM downloads][npm-downloads]][npm-url] [![Join Slack][slack-image]][slack-url]
+
+`neutrino-middleware-hot` is Neutrino middleware for enabling named modules output via `NamedModulesPlugin`.
```
Comment (helfi92) [inline docs/middleware/neutrino-middleware-named-modules/README.md]: s/`neutrino-middleware-hot`/`neutrino-middleware-named-modules`/

File: docs/presets/neutrino-preset-airbnb-base/README.md [hunk 4]
```diff
@@ -177,21 +184,35 @@ project.
 ### Advanced configuration
 
 By following the [customization guide](/customization/advanced.md) and knowing the rule and loader IDs above,
-you can override and augment the build by creating a JS module which overrides the config.
+you can override and augment the build by creating a JS module which overrides the config. This preset is also
+Neutrino middleware, making it easy to compose and extend the configuration.
```
Comment (helfi92) [inline docs/presets/neutrino-preset-airbnb-base/README.md]: insert "a" before Neutrino.

File: docs/usage.md [hunk 5]
```diff
@@ -14,16 +14,16 @@ In a typical project:
...
+typical recommendations for script target names. You may choose any different name if desired
 for your project.
 
 ## Building for development
 
 Neutrino provides the command `neutrino start` for creating a bundle during development. Using
 `neutrino start` sets the Node.js environment to `development` using the `NODE_ENV` environment variable,
 which is available in your project source code. Depending on the presets you are using, `neutrino start`
-may also spin up a development server with hot module reloading capabilities.
+may also spin up a development server with hot module replacement capabilities.
```
Comment (helfi92) [inline docs/usage.md]: I think hot module replacement is always capitalized.

File: docs/middleware/neutrino-middleware-banner/README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+The following is a list of plugins and their identifiers which can be overridden:
+
+- `banner`: Injects string content into application source code.
+
+## Contributing
+
+This preset is part of the [neutrino-dev](https://github.com/mozilla-neutrino/neutrino-dev) repository, a monorepo
+containing all resources for developing Neutrino and its core presets. Follow the
+[contributing guide](/contributing/README.md) for details.
```
Comment (eliperelman) [inline docs/middleware/neutrino-middleware-banner/README.md]: This is an issue with gitbook; I'll see if I can fix it.

**Pull request merged**